### PR TITLE
Update CA_QC.py

### PR DESCRIPTION
**Pull request by EarthlyAlien**
Updated CA_QC.py parser so that it can return multiple datapoints instead of just single recent datapoint.

### CODE REVIEW

File: parsers/CA_QC.py [hunk 1]
```diff
@@ -38,13 +38,13 @@ def if_exists(elem: dict, etype: str):
             return 0.0
 
     data = _fetch_quebec_production()
+    list_res = []
     for elem in reversed(data["details"]):
         if elem["valeurs"]["total"] != 0:
-
-            return {
+            list_res.append(dict({
```
Comment (jarek) [inline parsers/CA_QC.py:44]: `dict({})` is redundant - just `{}` also creates a dict (this is called a "dict literal"). Same on line 47 below

Comment (EarthlyAlien) [reply parsers/CA_QC.py]: Thanks @jarek. I've made changes accordingly.

**Review by madsnedergaard — APPROVED** [review]
Nice! 🙌

**Comment by VIKTORVAV99** [comment]
Looks like the python formatting check is failing.
If you run `poetry run format` that should resolve the issue.

**Comment by EarthlyAlien** [comment]
@VIKTORVAV99 I ran `poetry run format` but nothing changed.

Here's the screenshot after running the above command.

image

**Comment by VIKTORVAV99** [comment]
> @VIKTORVAV99 I ran `poetry run format` but nothing changed.
> 
> Here's the screenshot after running the above command.
> 
> image

It looks like black isn't installed yet. Try running `poetry install` first then `poetry run format` again.

**Review by EarthlyAlien — COMMENTED** [review]
Formatting done.

**Pull request merged**